indico_hub/register.py

```diff
@@ -0,0 +1,62 @@
+import requests
+import json
+import requests
+from requests.exceptions import HTTPError, RequestException, Timeout
+from werkzeug.urls import url_join
+COMMUNITY_HUB_URL = "https://hub.getindico.io"
```

Comment:
why is this hardcoded here? sure, this is a script to test, but it should access your local instance of this microservice, not the production community hub